yaer/runner.py

```diff
@@ -106,5 +106,10 @@ def run_experiments(experiments_to_run, regex_to_match,
                 logger.debug('Removing %s', no_time_path)
                 shutil.rmtree(no_time_path)
             dumpers.append(FileDumper(final_dump_path))
-        run_experiment(name, exp, dumpers)
+
+        try:
+            run_experiment(name, exp, dumpers)
+        except Exception:
+            logger.exception("Experiment internal error !")
```

Comment:
Hi Mariano!

What would happen if, for instance, a `ValueError` exception is raised inside an experiment. How we would know which exception was raised?

IMHO we should at least log the exception msg in order to know which was the internal error.

WDYT?

Reply:
Hi ! According to **exception** method documentation [1], info is included by default. Am I missing some relevant information ?

[1]: https://docs.python.org/3/library/logging.html#logging.Logger.exception